db/schema.rb

```diff
@@ -153,26 +153,27 @@
     t.index ["message_id"], name: "index_attachments_on_message_id"
   end
 
-  create_table 'audits', force: :cascade do |t|
-    t.bigint 'auditable_id'
-    t.string 'auditable_type'
-    t.bigint 'associated_id'
-    t.string 'associated_type'
-    t.bigint 'user_id'
-    t.string 'user_type'
-    t.string 'username'
-    t.string 'action'
-    t.jsonb 'audited_changes'
-    t.integer 'version', default: 0
-    t.string 'comment'
-    t.string 'remote_address'
-    t.string 'request_uuid'
-    t.datetime 'created_at'
-    t.index %w[associated_type associated_id], name: 'associated_index'
-    t.index %w[auditable_type auditable_id version], name: 'auditable_index'
-    t.index ['created_at'], name: 'index_audits_on_created_at'
-    t.index ['request_uuid'], name: 'index_audits_on_request_uuid'
-    t.index %w[user_id user_type], name: 'user_index'
+  create_table "audits", force: :cascade do |t|
+    t.bigint "auditable_id"
+    t.string "auditable_type"
+    t.bigint "associated_id"
+    t.string "associated_type"
+    t.bigint "user_id"
+    t.string "user_type"
+    t.string "username"
+    t.string "action"
+    t.jsonb "audited_changes"
+    t.integer "version", default: 0
+    t.integer "integer", default: 0
+    t.string "comment"
+    t.string "remote_address"
+    t.string "request_uuid"
+    t.datetime "created_at"
+    t.index ["associated_type", "associated_id"], name: "associated_index"
+    t.index ["auditable_type", "auditable_id", "version"], name: "auditable_index"
+    t.index ["created_at"], name: "index_audits_on_created_at"
+    t.index ["request_uuid"], name: "index_audits_on_request_uuid"
+    t.index ["user_id", "user_type"], name: "user_index"
```

Comment:
@sojan-official running the migration added this for me. Should I remove these changes from this patch?

Reply:
Removed it anyway, ignore